hamilton/execution/graph_functions.py

```diff
@@ -95,6 +96,29 @@ def combine_config_and_inputs(config: Dict[str, Any], inputs: Dict[str, Any]) ->
     return {**config, **inputs}
 
 
+def create_input_string(kwargs: dict) -> str:
+    """THis is a utility function to create a string representation of the inputs to a function.
+
+    This is useful for debugging, as it can be printed out to see what the inputs were.
+
+    :param kwargs: The inputs to the function that errored.
+    :return: The string representation of the inputs, truncated appropriately.
+    """
+    pp = pprint.PrettyPrinter(width=80)
+    inputs = {}
+    for k, v in kwargs.items():
+        item_repr = repr(v)
+        if len(item_repr) > 50:
+            item_repr = item_repr[:50] + "..."
+        else:
+            item_repr = v
+        inputs[k] = item_repr
+    input_string = pp.pformat(inputs)
+    if len(input_string) > 1000:
+        input_string = input_string[:80] + "..."
```

Comment:
Why 80 versus 1000? Shouldn't we truncate it to 1000?

Reply:
oops, yeah missed that.